README.md

```diff
@@ -113,7 +113,8 @@ production use, but `debug` is also available for more verbosity.
 
 There are a few options for enabling it:
 
-1. Set the environment variable `STRIPE_LOG` to the value `debug` or `info`
+1. Set the environment variable `STRIPE_LOG` to the value `debug`, `info` or
+   `warning`
```

Comment:
I couldn't tell that easily from the changes whether this already happens, but warnings should be emitted by default (if `STRIPE_LOG` isn't set), because otherwise no one will ever see them. Does the code do that right now?

Reply:
Could we have it respect either `warn` or `warning`? Especially in logging libraries these two are often interchanged that it'd be sort of a nice feature to support both. If we're supporting only one, I'd say that we might want to do `warn` instead.

Reply:
@brandur-stripe My code currently requires a log level to be set in order to show warnings. I just did this to follow the existing pattern from `info` and `debug` - I'm fine with changing it.

We could simplify the interface and not allow `warn` (or `warning`) at all, and just always print warnings. Is there a case (`STRIPE_LOG`/`stripe.log` value) where you wouldn't expect warnings to be printed?

Reply:
That sounds a whole lot simpler - I'll do that!